spec/takeaway_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "./lib/takeaway"
+require "./lib/menu"
```

Comment:
This test is not independent of the menu class - I don't think tests should require class files other than the one they are treating.  A double of menu should be used.